<launch>
 <group>
 	<remap from="face_recognizer_node/image_raw/compressed" to="camera_node/image_raw/compressed" />
 	<remap from="face_recognizer_node/image_raw" to="camera_node/image_raw" />
  <node name="face_recognizer_node" pkg="pi_cam" type="face_recognizer_node.py" output="screen" >
  </node>
 </group>
</launch>
